.github/workflows/latest-matrix-sdk-crypto.yml

```diff
@@ -0,0 +1,87 @@
+name: Build against latest matrix-sdk-crypto
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  '22 1 * * *'
```

Comment:
Can you add a comment explaining what it means, for the peasants like me who don't speak crontab fluently? :)

Reply:
Thanks, added in f5ec776